### PR TITLE
[KARAF-5447] Support Spring 5.0.x

### PR DESCRIPTION
**Pull request by gnodet**

### CODE REVIEW

**Comment by jbonofre** [comment]
I already started a PR about that. Let me check if we are sync.

**Comment by gnodet** [comment]
That one uses the newer servicemix bundles that fix the deployment problems.
The 5.0.0.RELEASE_1 bundles can't deploy beause there's an import on a package that has been removed in 5.0.0.

**Comment by jbonofre** [comment]
Got it. Thanks. I will tackle the bundles release and review the PR.

**Comment by jbonofre** [comment]
ServiceMix Bundles 2017.11 release is on vote, including Spring 5.0.2.RELEASE_1 bundles. Can you update the PR accordingly please ?

**Comment by jbonofre** [comment]
retest this please

**Comment by jbonofre** [comment]
Can you please rebase this PR (as some tests have been fixed on master) ?

File: pom.xml [hunk 1]
```diff
@@ -342,6 +343,10 @@
                 <enabled>true</enabled>
             </snapshots>
         </repository>
+		<repository>
```
Comment (jbonofre) [inline pom.xml]: It can be removed as now the bundles are on Central.

File: itests/pom.xml [hunk 2]
```diff
@@ -266,6 +266,7 @@
                         <spring41.version>${spring41.version}</spring41.version>
                         <spring42.version>${spring42.version}</spring42.version>
                         <spring43.version>${spring43.version}</spring43.version>
+                        <spring50.version>${spring50.version}</spring50.version>
```
Comment (jbonofre) [inline itests/pom.xml]: The `ci-build-profile` profile has to be updated as well else it fails on Jenkins.

**Comment by gnodet** [comment]
retest this please

**Comment by jbonofre** [comment]
`ci-build-profile` still doesn't contain the `spring50.version` property. It has to be there.

File: itests/pom.xml (hunk 2 above)
Comment (jbonofre) [reply itests/pom.xml]: This property needs to be in `ci-build-profile` as well.

**Comment by jbonofre** [comment]
retest this please

**Comment by gnodet** [comment]
retest this please

**Comment by jbonofre** [comment]
FYI, the `retest this please` comment is not required when you push an update on the PR branch: I configured Jenkins to automatically trigger a build.

**Pull request merged**